xcell/cls/cl.py

```diff
@@ -68,11 +68,11 @@ def get_dtypes(self):
 
 
 class Cl(ClBase):
-    def __init__(self, data, tr1, tr2):
-        super().__init__(data, tr1, tr2)
+    def __init__(self, data, tr1, tr2, ignore_existing_yml=False):
+        super().__init__(data, tr1, tr2, ignore_existing_yml)
```

Comment:
@carlosggarcia do you know why this PR is showing these as modifications? Shouldn't these be in master already?